shared-bin/src/logger.rs

```diff
@@ -113,9 +122,23 @@ pub fn init(
         None
     };
 
-    Registry::default()
-        .with(Layer::and_then(telemetry, fmt_layer).with_filter(filter))
+    let mut error_only = vec![
+        "Broadcast offers to taker",
```

Comment:
these strings are still really fragile - if you want to depend on particular string to disable spans, export it as `pub const` from the other crate. (applies to all span names you want to use here)